ap/tasks/reduction.py

```diff
@@ -76,30 +76,38 @@ def run(self):
             nano_file = input_file.load(formatter="uproot")
 
         # iterate over chunks of events and diffs
+        input_paths = [nano_file]
+        input_paths.append(inputs["sel"]["res"].path)
+        input_paths.extend([inp.path for inp in inputs["calib"]])
+        if self.selector_func.produced_columns:
+            input_paths.append(inputs["sel"]["cols"].path)
```

Comment:
See other comments regarding descriptive power of keyword arguments

```suggestion
            input_paths.append(inputs["selection"]["columns"].path)
```